src/main/java/com/jalasoft/jfc/model/PdfConverter.java

```diff
@@ -61,7 +61,7 @@ public Boolean convert(PdfParam pdf){
                 renderer = new PDFRenderer(documenttoimage);
             }
             for (int i = 0; i < pages; i++) {
-                pathName = pdf.getPathOuput().toString() + pdf.getNameFile() +
+                pathName = pdf.getOutputPathFile() + pdf.getOutputFileName() +
                         i + "." + pdf.getPdfFormatImage().toString();
                 if (pdf.getDpi() != 100) {
```

Comment:
try to avoid magic numbers, you can declare a variable, then assign **100** to that one.